cmake/onnxruntime_mlas.cmake

```diff
@@ -471,4 +471,7 @@ endforeach()
 set_target_properties(onnxruntime_mlas PROPERTIES FOLDER "ONNXRuntime")
 if (WIN32)
   target_compile_options(onnxruntime_mlas PRIVATE "/wd6385" "/wd4127")
+  if (onnxruntime_ENABLE_STATIC_ANALYSIS)
+    target_compile_options(${target_name} PRIVATE  "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:/analyze:stacksize 131072>")
```

Comment:
>target_name [](http://example.com/codeflow?start=29&length=11)

`${target_name}` is not set in this file
the "not CUDA" generator expression doesn't need to be specified if there are no CUDA files right?